libraries/AP_HAL_ChibiOS/hwdef/BrahmaF4/defaults.parm

```diff
@@ -0,0 +1,13 @@
+# setup for LEDs on chan5
+SERVO9_FUNCTION 120
+# setup SERIAL2 for RCIN
+SERIAL1_BAUD 115
+NTF_LED_TYPES 257
+NTF_LED_LEN 6
+SERVO_BLH_AUTO 1
```

Comment:
all the following lines are usually not set from their defaults unless this is a turnkey system